cocotb/_build_libs.py

```diff
@@ -137,16 +137,19 @@ def _gen_import_libs(self, def_dir):
                 )
 
 
-def _extra_link_args(lib_name):
+def _extra_link_args(lib_name=None, rpath=None):
     """
-    Add linker `install_name` argument on osx to load dependencies from the directory
-    where vpi/vhpi/fli library is located
+    Add linker argument to load dependencies from the directory where vpi/vhpi/fli library is located
+    On osx use`install_name`
+    On Linux use`rpath`
```

Comment:
```suggestion
    Use `rpath` on all platforms
```

Reply:
You are sure? On Windows there is rpath.

Reply:
Well, this matches the implementation, and what we were doing before.